addons/crm/crm_lead.py

```diff
@@ -730,28 +730,35 @@ def convert_opportunity(self, cr, uid, ids, partner_id, user_ids=False, section_
 
         return True
 
+    @api.one
```

Comment:
@api.one is indeed deprecated in v9.0. Moreover, the method create takes a dict as parameter, not a list of dicts: https://www.odoo.com/documentation/8.0/reference/orm.html#openerp.models.Model.create Since @api.one wraps the result in a list, that's not a good idea at all.

Finally, we avoid mixing both APIs in the same file. I agree that @api is used in order to be able to mix APIs, but the original idea is to be able to call a method written in new api from a method written in old api.